puppet/ha-puppet/screenshot.js

```diff
@@ -250,6 +288,7 @@ export class Browser {
             const panelResolver = mainEl.shadowRoot?.querySelector(
               "partial-panel-resolver",
             );
+            // noinspection JSUnresolvedReference
```

Comment:
what's this?

Reply:
Suppression comments from WebStorm. Removed!